neuro_san/test/evaluators/gist_agent_evaluator.py

```diff
@@ -128,6 +128,9 @@ def determine_pass_fail(self, answer: str) -> bool:
         test_passes: bool = passing and not failing
         test_fails: bool = failing and not passing
         only_one: bool = test_passes or test_fails
-        self.asserts.assertTrue(only_one)
+
+        # Specifically use assertEqual() here to reserve assertTrue/False for
+        # whether or not the test itself passed, as those feed into the Assessor.
+        self.asserts.assertEqual(only_one, True)
```

Comment:
This allows me to subvert the use of assertTrue() and assertFalse() elsewhere to collect Assessor test data.
For now this works but in a later PR I will come up with a better mechanism since it's so natural to want to use assertTrue/False.